server/routerlicious/packages/lambdas/src/utils/connectionCountLogger.ts

```diff
@@ -38,8 +38,6 @@ export class ConnectionCountLogger implements IConnectionCountLogger {
 			LumberEventName.TotalConnectionCount,
 		);
 		if (!this.cache || !this.cache.incr) {
-			connectionCountPerNodeMetric.error(`Redis Cache not found.`);
-			totalConnectionCountMetric.error(`Redis Cache not found.`);
```

Comment:
Do you want to log an info message instead, or is it just not needed at all?

Reply:
We don't need it right now. This feature is disabled even in production.